app/src/main/java/com/example/countryinfo/CountryDetailsDialog.kt

```diff
@@ -0,0 +1,42 @@
+package com.example.countryinfo
+
+import android.app.Dialog
+import android.content.DialogInterface
+import android.os.Bundle
+import androidx.appcompat.app.AlertDialog
+import androidx.fragment.app.DialogFragment
+import java.util.*
+
+class CountryDetailsDialog : DialogFragment() {
```

Comment:
[nit] Looks overcomplicated. Why not to build and show AlertDialog inside `DetailsFragment`?

Reply:
Removed.